### PR TITLE
Fix image parser

### PR DESCRIPTION
**Pull request by HydeHunter2**
Fixes a bug discussed by email.

### Changes
- The image parser becomes more secure.

----
UUID: af1477b6-509a-4611-b851-d7a8e3a9c02a

### CODE REVIEW

**Comment by HydeHunter2** [comment]
@crookedneighbor -- Hello again.
I tried to fix this bug by upgrading to the latest version of `markdown-it`, but It didn't help.

I'm sure that the bug appears here because if you remove the import of `markdown-it-linkify-images` from `habitica-markdown`, the bug disappears.

*This import:
https://github.com/HabitRPG/habitica-markdown/blob/2174d57581dfea9d8f44dde154f43bad6e787ff5/index.js#L17-L21

**Review by crookedneighbor — CHANGES_REQUESTED** [review]
Seems reasonable. Please add a test to verify this behavior.

File: index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ function markdownitLinkifyImages (md, config) {
     var imgClass = generateClass(config.imgClass)
 
     if (token.attrIndex('title') !== -1) {
-      title = ' title="' + token.attrs[token.attrIndex('title')][1] + '"'
+      title = ' title="' + md.utils.escapeHtml(token.attrs[token.attrIndex('title')][1]) + '"'
```
Comment (HydeHunter2) [inline index.js:18]: While writing the test, I found another bug in `title` attribute.
So thanks for the tip to write the test 👍

**Review by crookedneighbor — APPROVED** [review]

**Comment by crookedneighbor** [comment]
I’ll merge this and release it in a few days.

**Comment by crookedneighbor** [comment]
Released with v1.1.1

**Pull request merged**